core/src/test/java/com/linecorp/armeria/server/AnnotatedHttpServiceRequestConverterTest.java

```diff
@@ -755,54 +753,6 @@ public void testDefaultRequestConverter_bean3() throws Exception {
         assertThat(response.content().toStringUtf8()).isEqualTo(expectedResponseContent);
     }
 
-    @Test
-    public void testDefaultRequestConverter_beanDefineError() throws Exception {
```

Comment:
Need to add these cases tomorrow

Reply:
Moved to https://github.com/line/armeria/pull/1191/files#diff-acfda6b218745abeaece7a779cd740b7R35